Farsight RPG Community Edition/sheet.json

```
@@ -0,0 +1,8 @@
+{
+    "html": "sheets.html",
+    "css": "sheets.css",
+    "authors": "Vincenzo Aucello",
+    "roll20userid": "1376005",
+    "preview": "sheet_preview.png",
+    "instructions": "This Farsight RPG Sheet is made by its member community to work with its system. Every input is like the original pdf version and instrucions to fill it can be found in the manual of the game."
```

You're missing the legacy tag, see any other sheet.json for comparison. See [CSE](https://wiki.roll20.net/CSE) for info in what it represents.
```suggestion
"instructions": "This Farsight RPG Sheet is made by its member community to work with its system. Every input is like the original pdf version and instrucions to fill it can be found in the manual of the game.",
"legacy": true
```
is probably the right thing, as you've likely made the sheet without making it CSE-compliant.